cinema/serializers.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
```

Comment:
Consider using ModelSerializer instead of Serializer for the MovieSerializer class. ModelSerializer automatically generates fields based on the model and can simplify the code.